src/views/splash/splash.jsx

```diff
@@ -215,11 +215,11 @@ var Splash = injectIntl(React.createClass({
                 <Box
                         key="curator_top_projects"
                         title={
-                            'Projects Curated by ' +
+                            formatMessage({id: 'splash.projectsCuratedBy', defaultMessage: 'Projects Curated by '}) +
```

Comment:
ahh ok – that's probably an old string that I have failed to update to the new style we use. Thanks for the heads up that we should update that!